### ISSUE TITLE
Make cropper frame resizable

### ISSUE THREAD

**Issue by ilyastam** [jeduan/cordova-plugin-crop#4]
Hi, thank you for the great plugin.
I am using it to crop images to squares in my app, and I've noticed that user can change the cropper frame from default square to a rectangular. It would be nice to prevent user from being able to do that. 
It would be very useful if I could specify the width and height of a destination image and have cropper honor this aspect ratio.

**Comment by jeduan** [comment]
is that on Android?

**Comment by oleksiy-nesterov** [comment]
Yep, Android. No way to change ratio of cropping. Under Android this is always square. 
Is it possible? to add some options for plugin, to make possibility to control it.

**Comment by oleksiy-nesterov** [comment]
ops. Last version has fixed square ratio for IOS (( let's add some option to control this, always use square is not good.

**Comment by oleksiy-nesterov** [comment]
added modified version with "keepingAspectRatio" option
https://www.npmjs.com/package/cordova-plugin-crop-mod

**Comment by jeduan** [comment]
@oleksiy-nesterov why not sending a PR? just curious

**Comment by oleksiy-nesterov** [comment]
I thought, you do not support this plugin, because all my previous messages without answers/suggestions. Will do PR.

**Comment by jeduan** [comment]
You can become a maintainer if you're inclined.

**Comment by rajnandan1** [comment]
@oleksiy-nesterov  does not work when the source type is  PHOTOLIBRARY;

**Comment by ahmadramadan** [comment]
+1
